dataland-frontend/tests/e2e/specs/lksg/DataIntegrity.ts

```diff
@@ -51,6 +74,7 @@ describeIf(
           );
           cy.get("h1").should("contain", testCompanyName);
           uploadLksgDataViaForm();
+          validateFormUploadedData(storedCompany.companyId);
         });
     });
 
```

Comment:
The test in the following lines could be a component test.

Based on our team rule "If we touch an e2e test file, we migrate tests to component tests if possible" we will need to do that here.

Reply:
Yes. Will do that as soon as I can.

Reply:
No, won't do. DynamicDialog does not play well with component tests.

Reply:
Yes I see.
Let's leave it for now then, and keep in mind to discuss this problem with the team:

a) We cannot write component tests for stuff which includes a FormKit to be mounted
b) We cannot write component tests for stuff which includes a PrimeVue DynamicDialog to open up

These are pretty bad limitations in my opinion.  It excludes a lot of features from being able to component-test them.

Closing this now due to the lack of possiblity to do smth here.

Reply:
Should we open a backlog item to fix these limitations?

Reply:
HAve you done it? Then I can close this

Reply:
Yup

Reply:
👍  thx